tests/Doctrine/Tests/DBAL/Functional/Driver/SQLAnywhere/ConnectionTest.php

```diff
@@ -16,14 +17,16 @@ protected function setUp()
 
         parent::setUp();
 
-        if (! $this->_conn->getDriver() instanceof Driver) {
-            $this->markTestSkipped('sqlanywhere only test.');
+        if ($this->connection->getDriver() instanceof Driver) {
+            return;
         }
+
+        $this->markTestSkipped('sqlanywhere only test.');
```

Comment:
We don't seem to be consistent with using `$this` vs. `self`. But that can be fixed separately.

Reply:
I don't have a problem fixing it but let's make it part of the standard first.